app/controllers/approvers_controller.rb

```diff
@@ -8,47 +8,6 @@
 Generated by: https://github.com/swagger-api/swagger-codegen.git
 
 =end
-class ApproversController < ApplicationController
+class ApproversController < AdminsController
```

Comment:
Approver has more privilege than admin?

Reply:
but `ApproversController` is a subclass of `AminsController`, which seems opposite to what you claimed.

You may extract common code into a mixin to avoid this confusion.

Reply:
I created mixins for user and approver.